test/user.test.js

```diff
@@ -0,0 +1,245 @@
+const request = require('supertest');
+const expect = require('chai').expect;
+
+const app = require('../index');
+const fixture = require('../fixtures/user');
+
+describe('User controller', () => {
+  before(async () => {
+    await fixture();
+  });
+
+  describe('GET /api/users/:id', () => {
```

Comment:
This text should be like a story, without many technical details, but rather a sentence in english. I would organise the test like this:
```
describe('CRUD user', () => {
  describe('get user', () => {
    describe('when an existing user is fetched', () => {
      it('returns the user', () => {
      });
    });

    describe('when fetching a deleted user', () => {
      it('returns 404', () => {
      });
    });

    describe('when fetching a user that does not exist', () => {
      it('returns 404', () => {
      });
    });
  });
});
```
I hope you get the idea.